sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/csv/CSVOptions.scala

```diff
@@ -149,7 +149,12 @@ class CSVOptions(
     parameters.getOrElse(DateTimeUtils.TIMEZONE_OPTION, defaultTimeZoneId))
 
   // A language tag in IETF BCP 47 format
-  val locale: Locale = parameters.get(LOCALE).map(Locale.forLanguageTag).getOrElse(Locale.US)
+  val locale: Locale = try {
+    parameters.get(LOCALE).map(Locale.forLanguageTag).getOrElse(Locale.US)
+  } catch {
+    case _: NullPointerException =>
```

Comment:
Can we check if the value is `null`, and throw an exception instead?

Reply:
and this issue should exist in almost all other options. Would probably be best to fix them together